parl/remote/tests/send_job_test.py

```diff
@@ -44,9 +44,9 @@ def test_send_file(self):
         worker = Worker('localhost:{}'.format(port), 1)
         time.sleep(2)
 
-        os.system('mkdir ./rom')
-        os.system('touch ./rom/pong.bin')
-
+        os.system('mkdir ./rom_files')
+        os.system('touch ./rom_files/pong.bin')
+        assert os.path.exists('./rom_files/pong.bin')
```

Comment:
What I mean in previous comment is that we have to check the existence of files in `client.py` before distributing them.

Reply:
Add ``assert os.path.exists(...)`` before create the client.

And add another test in ``send_job_test.py`` to check when the distributed file does not exist.